packaging/helm/skipper-aws/templates/skipper/service-internal.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: skipper-internal
+  namespace: kube-system
+  labels:
+    application: skipper-ingress
+spec:
+  type: ClusterIP
+  clusterIP: {{ .Values.skipperSvcIP }}
```

Comment:
Do we need to specify this? Wouldn't it be better to let Kubernetes auto-assign it? (Or make it optional)

Reply:
No, because you need to add this to the CoreDNS template, too.
If you have 100 clusters and the IP is dynamically created, you have to change the Corefile for all of your clusters.